### DISCUSSION TITLE
Since Tabby 0.9: Subscription needed

### DISCUSSION THREAD

**Discussion by rettelbachj** [TabbyML/tabby#1686] [General]
**Describe the bug**
I just updated TabbyML to 0.9 and I can't add new members since a subscription is neede.
Unfortunately there is no information how to get a subscription. Also I would like to point that it would have been nice to announce, that the service won't be free in the next i.e. in the Release Notes. Or did I just miss the announcement?
Still I want to say thank you for the great work you put in this project!

**Comment by wsxiaoys** [comment]
Inviting users should remain accessible for a community license, provided that you have fewer than five users. A team or enterprise license is mandatory for instances with more than five users. 

To purchase the license, you may use the "Schedule a Demo Call" link on the subscription page.

**Reply by traverseda** [reply]
This is a pretty strange limitation for the open source, community oriented, build. Is the community edition including enterprise code? Is there a way to run tabby without any of the EE modules?

**Comment by hinricht** [comment]
The lack of an announcement is indeed troubleing, please try to be as transparent as possible about this limitation, i.e. a blog article. Thanks for this project anyhow !

**Reply by Starefossen** [reply]
Second it, it has been well over a month and still no word on the website. I am not arguing about the introduction of a subscription service, but rather the lack of information about it.

**Comment by wsxiaoys** [comment]
> The lack of an announcement is indeed troubleing, please try to be as transparent as possible about this limitation, i.e. a blog article.

Sure, as it is a new version being released, the documentation is currently lacking. We are working hard to improve it. If this is currently blocking your workflow, consider staying on version 0.8 for now.

>  Thanks for this project anyhow !

Thank you too, and thank you for understanding. :)

**Comment by rettelbachj** [comment]
Ok we will stay on 0.8.3 then! :) Thanks a lot for answering and understanding!
I have another question, would it also be a feasible option to simply share an authentification token across multiple users or are the code suggestions dependent on previous snippets with the same token?
